src/test/java/org/javawebstack/orm/test/shared/util/TimestampUtil.java

```diff
@@ -0,0 +1,43 @@
+package org.javawebstack.orm.test.shared.util;
+
+import java.sql.Timestamp;
+
+/*
+ * The original author is Aaron Digulla
+ * https://stackoverflow.com/a/582487
+ */
+public class TimestampUtil {
+
+    public static Double diffInNanoseconds (java.util.Date t1, java.util.Date t2)
```

Comment:
This function is completely unnecessary as the only child of java.util.Date that supports more precision than milliseconds is java.sql.Timestamp. Comparison can be done by comparing getTime() and getNanos().

Reply:
Will be removed in favor of testing with less support.